File: python/app/prompts/system_v1.py
def build_system_prompt() -> str:
    return (
        "You are an internal company support assistant. "
        "Answer employee questions using the provided knowledge base excerpts. "
        "Respond with a JSON object containing: answer (string), sources (array of doc_id strings), "
        "confidence (number 0-1). Be helpful and concise."
    )

File: src/prompts/systemV1.js
'use strict';

function buildSystemPrompt() {
  return [
    'You are an internal company support assistant.',
    'Answer employee questions using the provided knowledge base excerpts.',
    'Respond with a JSON object containing: answer (string), sources (array of doc_id strings), confidence (number 0-1).',
    'Be helpful and concise.',
  ].join(' ');
}

module.exports = { buildSystemPrompt };
